src/cascadia/TerminalApp/Command.idl

```diff
@@ -12,5 +12,7 @@ namespace TerminalApp
         String Name;
         ActionAndArgs Action;
         String KeyChordText;
+
+        String IconString;

```

Comment:
nit: can we make this `IconPath` to align with `Tab`?

Reply:
I was actually thinking of leaving this as IconString in case we wanted to use font icon glyphs for some commands alongside icon paths for other commands/tab switcher actions.

I was trying to figure out a way to allow both font icons and icon paths to be used here (like described in the issue you filed #6644) and so I made a converter that would _ideally_ be able to take in a string and figure out if it's a hex to a glyph or it's a path to an icon, and return a FontIconSource/BitmapIconSource depending on what string it got.

Reply:
I wonder if the right thing to do here is actually to use [`IconSource`](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.iconsource?view=winrt-19041), which encapsulates bitmap/font/path/symbol in one handy wrapper

Reply:
There's a couple downsides to IconSource. One: coupling Command (datamodel) and XAML stuff. Two: it doesn't let you provide a loaded bitmap object, and i cannot possibly imagine why

Reply:
but.. neither does String :smile:

Reply:
This is definitely the better way to do it! 👍

Reply:
Also means I can get rid of the converter even though it's so cool 😢